kolibri/plugins/coach/assets/src/vue/exercise-detail-view/index.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+
+  <div id="whole-page">
+    <div id="top-bar">
+      <router-link id="back-btn" :to="backPageLink">
+        <mat-svg
+          class="back svg-back"
+          category="navigation"
+          name="arrow_back"
+        />
+        <p class="back">
+          <slot name="text"/>
+        </p>
+      </router-link>
+    </div>
+    <div id="page-body">
+      <slot name="body"/>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+    props: {
+      backPageLink: {
+        type: Object,
+        required: true,
+      }
+    },
+  };
+
+</script>
+
+
+<style lang="stylus" scoped>
+
+  @require '~kolibri.styles.definitions'
+
+  #whole-page
+    left: 0
+    top: 0
+    z-index: 100
```

Comment:
arbitrary z-index? let's follow the elevations here: https://material.io/guidelines/material-design/elevation-shadows.html (cc @christianmemije )